R/evaluation.R

```diff
@@ -0,0 +1,24 @@
+#' Queries an evaluation queue for submissions
+#'
+#' @param syn A Synapse object
+#' @param uri A URI for evaluation queues (select * from evaluation_12345)
+#'
+#' @return The leaderboard results in a tibble.
+#' @examples
+#' library(challengerutils)
+#' use_condaenv('challenge')
+#' synapseclient <- import('synapseclient')
+#' syn <- synapseclient$Synapse()
+#' syn$login()
+#' leaderboard_results <- evaluation_queue_query(syn, 'select * from evaluation_12345')
+#' @export
+evaluation_queue_query <- function(syn, query) {
+  challengeutils <- import('challengeutils')
+  query_func <- challengeutils$utils$evaluation_queue_query
+  leaderboard <- query_func(syn = syn, uri = query)
+  vals_gen <- reticulate::iterate(leaderboard)
+  vals <- lapply(vals_gen, function(x){
+    x %>% unlist %>% as.data.frame() %>% t() %>% as_tibble()
```

Comment:
`%>%` operator isn't readily available; we will need to let users know to also load `tidyverse` or `magrittr`

Reply:
It automatically loads when you install the package.  I load `dplyr`.  Should I load `magrittr` instead?

Reply:
Nah, `dplyr` should be fine.